.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "shared-modules"]
 	path = shared-modules
 	url = https://github.com/flathub/shared-modules.git
-[submodule "wrapper"]
-	path = wrapper
-	url = https://github.com/Lctrs/jetbrains-flatpak-wrapper.git
+[submodule "ide-flatpak-wrapper"]
+	path = ide-flatpak-wrapper
+	url = https://github.com/flathub/ide-flatpak-wrapper.git
```

Comment:
Please don't. flathub/ide-flatpak-wrapper is tailored to vscode's related IDEs. We would need to at least override first_run.txt

Reply:
actually we do not use first_run.txt

the actual version are ide agnostic, just remain some naming to it

Reply:
We do use it on first run. I'll probably turn my wrapper into holding just the first_run.txt to use for jetbrains products.

Reply:
using `-Dfirst_run_template` we can pass our file